src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -0,0 +1,22 @@
+$color-picker-control-height: 48px;
+
+@import '../styles';
+
+.bp-ColorPickerControl {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $color-picker-control-height;
+    height: $color-picker-control-height;
+}
+
+.bp-ColorPickerControl-button {
+    @include bp-ControlButton;
+
+    svg {
```

Comment:
We should avoid styling via tag name, when possible.

Reply:
I'll add a class for it.